src/common/streak.ts

```diff
@@ -0,0 +1,73 @@
+import { DayOfWeek, DEFAULT_WEEK_START } from './date';
+import {
+  Day,
+  FREEZE_DAYS_IN_A_WEEK,
+  STREAK_RECOVERY_MAX_GAP_DAYS,
+} from './users';
+
+interface DaysProps {
+  day: number;
+  firstDayOfWeek: Day;
+  lastDayOfWeek: Day;
+  lastView: Date;
+}
+
+const getNextDay = (day: number) => (day === 6 ? 0 : day + 1);
+const getPreviousDay = (day: number) => (day === 0 ? 6 : day - 1);
+
+export const getAllowedDays = ({
+  day: dayToday,
+  firstDayOfWeek,
+  lastDayOfWeek,
+  lastView,
+}: DaysProps) => {
+  // do something
```

Comment:
Why is this here?

Reply:
Just forgot to remove when I was writing the function. Should've been removed by now.